requirements.txt

```diff
@@ -0,0 +1,25 @@
+future
+sqlalchemy
+
+PyPDF2
+babel
+blinker
+click
+flask
+flask_babel
+flask_login
+flask_principal
+iso-639
+itsdangerous
+jinja2
+markupsafe
+pytz
+requests
+singledispatch
+six
+sqlalchemy
+tornado
+https://pypi.python.org/packages/02/f8/97105237d0ba693b6f0bdcd94da0504e9a4433988c4393d8d3049094be7a/validate-1.0.1.tar.gz
```

Comment:
pip repository not available since the module is too old , and new one is changed to ConfigObj submodule
pip repository validator.py is not the exactly module indeed.